README.md

```diff
@@ -1,16 +1,16 @@
 # pdc-backend
 
-Trata-se de uma API REST que tem a função de se comunicar com o banco de dados através de inserções e consultas e servir esses dados para a aplicação frontend.
+Trata-se de um serviço que tem a função de prover a infraestrutura de banco de dados. Aqui o banco de dados é criado, tabelas e relacionamentos são definidos e, por fim é populado com os dados, então esse banco de dados é disponibilizado para que uma api possa acessá-lo.
 
 A aplicação utiliza do Sequelize para conexão e acesso ao banco de dados, ele nada mais é do que um ORM (Object Relational Mapping), onde as tabelas do banco de dados são definidas através de migrations e classes e não é necessário se preocupar com comandos na linguagem SQL, porém, estes ainda podem ser utilizados.
 
 ### Configurações do docker e postgreSQL
 
-Antes de tudo, é necessário ter o docker e o postgres instalados. 
+Antes de tudo, é necessário ter o docker e o postgres instalados na máquina virtual, e também na máquina do usuário caso o mesmo deseje testar localmente. 
 
-Para instalar o docker em distribuições linux ubuntu, siga [aqui](https://docs.docker.com/engine/install/ubuntu/).
+Para instalar o docker em distribuições linux ubuntu, siga [aqui](https://docs.docker.com/engine/install/ubuntu/){:target="_blank" rel="noopener"}.
 
-Para instalar o postgres (também no ubuntu) em sua versão 10, basta seguir o [gist](https://gist.github.com/paulojuniore/722a2d46814363f85ae526e800f2303d). 
+Para instalar o postgres (também no ubuntu) em sua versão 10, basta seguir o [gist](https://gist.github.com/paulojuniore/722a2d46814363f85ae526e800f2303d){:target="_blank" rel="noopener"}. 
```

Comment:
O primeiro comando dessas instruções está falhando:

sudo apt -y install postgresql-10 postgresql-contrib postgresql-server-dev-10
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package postgresql-10
E: Unable to locate package postgresql-server-dev-10

Reply:
O segundo sudo apt-get update gera esse erro:

sudo apt-get update
Ign:1 http://apt.postgresql.org/pub/repos/apt/lsb_release -cs-pgdg InRelease
Ign:2 http://apt.postgresql.org/pub/repos/apt/lsb_release -cs-pgdg Release           
Hit:3 https://download.docker.com/linux/ubuntu xenial InRelease                      
Ign:4 http://apt.postgresql.org/pub/repos/apt/lsb_release -cs-pgdg/main amd64 Packages                                                
Ign:5 http://apt.postgresql.org/pub/repos/apt/lsb_release -cs-pgdg/main all Packages                                                  
Ign:4 http://apt.postgresql.org/pub/repos/apt/lsb_release -cs-pgdg/main amd64 Packages                     
Ign:5 http://apt.postgresql.org/pub/repos/apt/lsb_release -cs-pgdg/main all Packages 
Ign:4 http://apt.postgresql.org/pub/repos/apt/lsb_release -cs-pgdg/main amd64 Packages
Ign:5 http://apt.postgresql.org/pub/repos/apt/lsb_release -cs-pgdg/main all Packages 
Ign:4 http://apt.postgresql.org/pub/repos/apt/lsb_release -cs-pgdg/main amd64 Packages
Ign:5 http://apt.postgresql.org/pub/repos/apt/lsb_release -cs-pgdg/main all Packages 
Ign:4 http://apt.postgresql.org/pub/repos/apt/lsb_release -cs-pgdg/main amd64 Packages
Ign:5 http://apt.postgresql.org/pub/repos/apt/lsb_release -cs-pgdg/main all Packages 
Err:4 http://apt.postgresql.org/pub/repos/apt/lsb_release -cs-pgdg/main amd64 Packages
  404  Not Found [IP: 147.75.85.69 80]
Ign:5 http://apt.postgresql.org/pub/repos/apt/lsb_release -cs-pgdg/main all Packages 
Hit:6 http://nova.clouds.archive.ubuntu.com/ubuntu xenial InRelease                  
Hit:7 http://security.ubuntu.com/ubuntu xenial-security InRelease                    
Hit:8 http://nova.clouds.archive.ubuntu.com/ubuntu xenial-updates InRelease
Hit:9 http://nova.clouds.archive.ubuntu.com/ubuntu xenial-backports InRelease
Reading package lists... Done                     
W: The repository 'http://apt.postgresql.org/pub/repos/apt/lsb_release -cs-pgdg Release' does not have a Release file.
N: Data from such a repository can't be authenticated and is therefore potentially dangerous to use.
N: See apt-secure(8) manpage for repository creation and user configuration details.
E: Failed to fetch http://apt.postgresql.org/pub/repos/apt/lsb_release/dists/-cs-pgdg/main/binary-amd64/Packages  404  Not Found [IP: 147.75.85.69 80]
E: Some index files failed to download. They have been ignored, or old ones used instead.